vars/envToPragmas.groovy

```diff
@@ -0,0 +1,18 @@
+/* groovylint-disable ParameterName */
+// vars/envToPragmas.groovy
+
+/**
+ * envToPragmas.groovy
+ *
+ * envToPragmas variable
+ */
+
+Map call() {
+    Map pragmas = [:]
+    if (env.pragmas)
+        pragmas = "${env.pragmas}"[1..-2].split(', ').collectEntries { entry ->
+            String[] pair = entry.split('=', 2)
+            [(pair.first()): pair.last()]
```

Comment:
The value is now trimmed when parsing the commit message. So no need to trim here. And no need to match on `= ` anymore.